benchmark/common.js

```diff
@@ -36,6 +37,15 @@ if (module === require.main) {
   runBenchmarks();
 }
 
+function hasWrk() {
+  var result = child_process.spawnSync('wrk', ['-h']);
+  if (result.error.code === 'ENOENT') {
+    console.error('Couldnt locate `wrk` which is needed for running ' +
```

Comment:
`'couldn\'t'`?

Also, this might still be better of being more detailed. Point people to the benchmarks readme maybe?